modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/GroupUpdateRequest.java

```diff
@@ -272,19 +302,28 @@ private static CompletableFuture<Integer> partitionUpdate(
                     null
             );
         } else {
-            Set<Assignment> stableAssignments = Set.copyOf(partAssignments);
+            assert !partAssignments.isEmpty() : "Alive nodes with data should not be empty";
+
+            List<Assignment> nextAssignments = new ArrayList<>(partAssignments);
+
+            nextAssignments.sort(
```

Comment:
Hmm, not sure that I get it. It's required to choose the node with max lastLogId as forced one.

Reply:
Oops, I've missed `.thenComparing(Assignment::consistentId));`